pkg/java/pom/pom.go

```diff
@@ -296,6 +308,38 @@ func (props *properties) UnmarshalXML(d *xml.Decoder, start xml.StartElement) er
 	return nil
 }
 
+func (deps *pomDependencies) UnmarshalXML(d *xml.Decoder, _ xml.StartElement) error {
+	*deps = pomDependencies{}
```

Comment:
IIUC, we don't need this line.

Reply:
Right. Thanks! 
Changed in https://github.com/aquasecurity/go-dep-parser/pull/287/commits/522089e086271f85a67ca5d9dd61621d932eeff4